config/storage/middleware/stores/patching.json

```
@@ -30,6 +30,15 @@
           ]
         }
       }
+    },
+    {
+      "comment": "Handles the patching of specific Patch types",
```

```suggestion
      "comment": "Dedicated handlers that apply specific types of patch documents",
```
(the types are not being patched 🙂)